cupy/random/distributions.py

```diff
@@ -239,7 +239,7 @@ def gumbel(loc=0.0, scale=1.0, size=None, dtype=float):
        f(x) = \\frac{1}{\\eta} \
            \\exp\\left\\{ - \\frac{x - \\mu}{\\eta} \\right\\} \
            \\exp\\left[-\\exp\\left\\{-\\frac{x - \\mu}{\\eta} \
-           \\right\\}\\right],
+           \\right\\}\\right].
```

Comment:
In only here, I think comma `,` should be used. Because the sentence `where...` comes next.
Or it may be better to remove the `where...` sentence.

Reply:
Sorry.  I changed this line by an error.  I'll revert it.